.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=lock
```

Comment:
Is there a reason this is `lock` and not `false`?

Reply:
Mistake!